ci/deploy_stf.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: "Deploy STF"
+  hosts: controller
+  tasks:
+    - name: "Set the sto_dir if it isn't already set"
+      ansible.builtin.set_fact:
+        sto_dir: '{{ ansible_env.HOME }}/{{ zuul.project.src_dir }}'
+      when: sto_dir | default('') | length == 0
+
+    - name: "Get vars common to all jobs"
+      ansible.builtin.include_vars:
+        file: "vars-zuul-common.yml"
+
+    - name: "Get scenario-specific vars"
+      ansible.builtin.include_vars:
+        file: "vars-{{ scenario }}.yml"
```

Comment:
It is always set, the in a concrete job.

The first one is added in:
https://github.com/infrawatch/service-telemetry-operator/pull/480/files#diff-108978819c05ae183d88ec87959c2341a94cfc3f9465e3aeee82d554217b4f58R37

Reply:
If there's a default, we may end up in a case where the var is not set explicitly and we don't notice because the it defaults.

IMHO, not having a default means that we avoid the wrong value being set, but also get an early fail if the var is omitted. The error for an undefined var is pretty explicit